apps/expo/src/app/activity.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Text, View } from "react-native";
+import { AntDesign } from "@expo/vector-icons";
+
+// Props specific for the Activity component
+interface ActivityProps {
+  name: string;
+  location?: string;
+  attending?: boolean;
+  activityHeight?: number;
+}
```

Comment:
use RouterOutputs from utils/api.tsx to get the Activity type that is returned by the getSchedule procedure